src/branding/bazzite.scss

```diff
@@ -0,0 +1,38 @@
+@use "standard.scss" as *;
+
+:root {
+  /*
+  Brand colors
+  */
+  --brand-primary: rgb(138, 42, 226);
+  --brand-secondary: rgb(0, 71, 171);
+  --brand-text: white;
+
+  /*
+  Space between the logo and text; default is 0.5em, which is half a character wide
+  --logo-gap: 0.5em;
+  */
+
+  /*
+  Size of the logo
+  */
+  --logo-height: 3rem;
+  --logo-width: 3rem;
```

Comment:
I should probably have this have a default, and only probably specify width (with an override for height).